!Command
command:
- biom
- show-install-info
- format.org
positional:
- !Positional
  description: ''
  position: 0
  name: biom
  optional: false
- !Positional
  description: ''
  position: 1
  name: show-install-info
  optional: false
named: []
parent:
subcommands: []
usage: []
help_flag:
usage_flag:
version_flag:
help_text: "/usr/local/lib/python2.7/site-packages/pandas/_libs/__init__.py:4: RuntimeWarning:\
  \ numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got\
  \ 88\n  from .tslib import iNaT, NaT, Timestamp, Timedelta, OutOfBoundsDatetime\n\
  /usr/local/lib/python2.7/site-packages/pandas/__init__.py:26: RuntimeWarning: numpy.dtype\
  \ size changed, may indicate binary incompatibility. Expected 96, got 88\n  from\
  \ pandas._libs import (hashtable as _hashtable,\n/usr/local/lib/python2.7/site-packages/pandas/core/dtypes/common.py:6:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  from pandas._libs import algos, lib\n/usr/local/lib/python2.7/site-packages/pandas/core/util/hashing.py:7:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  from pandas._libs import hashing, tslib\n/usr/local/lib/python2.7/site-packages/pandas/core/indexes/base.py:7:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  from pandas._libs import (lib, index as libindex, tslib\
  \ as libts,\n/usr/local/lib/python2.7/site-packages/pandas/tseries/offsets.py:21:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  import pandas._libs.tslibs.offsets as liboffsets\n/usr/local/lib/python2.7/site-packages/pandas/core/ops.py:16:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  from pandas._libs import algos as libalgos, ops as libops\n\
  /usr/local/lib/python2.7/site-packages/pandas/core/indexes/interval.py:32: RuntimeWarning:\
  \ numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got\
  \ 88\n  from pandas._libs.interval import (\n/usr/local/lib/python2.7/site-packages/pandas/core/internals.py:14:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  from pandas._libs import internals as libinternals\n/usr/local/lib/python2.7/site-packages/pandas/core/sparse/array.py:33:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  import pandas._libs.sparse as splib\n/usr/local/lib/python2.7/site-packages/pandas/core/window.py:36:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  import pandas._libs.window as _window\n/usr/local/lib/python2.7/site-packages/pandas/core/groupby/groupby.py:68:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  from pandas._libs import (lib, reduction,\n/usr/local/lib/python2.7/site-packages/pandas/core/reshape/reshape.py:30:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  from pandas._libs import algos as _algos, reshape as _reshape\n\
  /usr/local/lib/python2.7/site-packages/pandas/io/parsers.py:45: RuntimeWarning:\
  \ numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got\
  \ 88\n  import pandas._libs.parsers as parsers\n/usr/local/lib/python2.7/site-packages/pandas/io/pytables.py:50:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  from pandas._libs import algos, lib, writers as libwriters\n\
  /usr/local/lib/python2.7/site-packages/h5py/__init__.py:36: RuntimeWarning: numpy.dtype\
  \ size changed, may indicate binary incompatibility. Expected 96, got 88\n  from\
  \ ._conv import register_converters as _register_converters\n/usr/local/lib/python2.7/site-packages/h5py/__init__.py:45:\
  \ RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility.\
  \ Expected 96, got 88\n  from . import h5a, h5d, h5ds, h5f, h5fd, h5g, h5r, h5s,\
  \ h5t, h5p, h5z\n/usr/local/lib/python2.7/site-packages/h5py/_hl/group.py:22: RuntimeWarning:\
  \ numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got\
  \ 88\n  from .. import h5g, h5i, h5o, h5r, h5t, h5l, h5p\nUsage: biom show-install-info\
  \ [OPTIONS]\nTry \"biom show-install-info -h\" for help.\n\nError: Got unexpected\
  \ extra argument (format.org)\n"
generated_using: []
